app/javascript/packs/Restaurants.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types'
+import Restaurant from './Restaurant'
+
+class Restaurants extends React.Component {
+
+    render() {
+        return (
+            <div>
+                { this.props.restaurants &&
+                    <div className="rest">
+                        {this.props.restaurants.map(function (restaurant) {
```

Comment:
The convention here is using an arrow function

Reply:
Changed. thanks!